community/kernel/src/test/java/org/neo4j/kernel/impl/api/integrationtest/KernelIT.java

```diff
@@ -61,41 +61,6 @@
 {
     // TODO: Split this into area-specific tests, see PropertyIT.
 
-    /**
-     * While we transition ownership from the Beans API to the Kernel API for core database
-     * interactions, there will be a bit of a mess. Our first goal is an architecture like this:
-     * <p>
-     * Users
-     * /    \
-     * Beans API   Cypher
-     * \    /
-     * Kernel API
-     * |
-     * Kernel Implementation
-     * <p>
-     * But our current intermediate architecture looks like this:
-     * <p>
-     * Users
-     * /        \
-     * Beans API <--- Cypher
```

Comment:
i remove it because i do not think it's a good place to describe produce design in a unit test :), but if you think it still make sense to somebody i can restore part that does not mention JTA ?